packages/edit-site/src/store/actions.js

```diff
@@ -144,7 +145,7 @@ export const removeTemplate =
 				sprintf(
 					/* translators: The template/part's name. */
 					__( '"%s" deleted.' ),
-					template.title.rendered
```

Comment:
This is not related to this PR, but I believe we'll land it soon and it's a small change.